src/xhr.js

```diff
@@ -144,6 +152,28 @@ class Xhr {
   }
 
   /**
+   * Fetches and returns the `FetchResponse` object.
+   *
+   * See https://developer.mozilla.org/en-US/docs/Web/API/GlobalFetch/fetch
+   *
+   * See `fetchAmpCors_` for more detail.
+   *
+   * @param {string} input
+   * @param {?FetchInitDef=} opt_init
+   * @return {!Promise<!FetchResponse>}
```

Comment:
Why response type `FetchResponse`? Let's just do `!Promise<!Document>`. This will be consistent with `fetchJson`